core/src/main/java/org/geysermc/geyser/item/type/PlayerHeadItem.java

```diff
@@ -46,8 +46,14 @@ public PlayerHeadItem(Builder builder, Block block, Block... otherBlocks) {
     public void translateComponentsToBedrock(@NonNull GeyserSession session, @NonNull DataComponents components, @NonNull TooltipOptions tooltip, @NonNull BedrockItemBuilder builder) {
         super.translateComponentsToBedrock(session, components, tooltip, builder);
 
+        // Get the rarity of the item or default to uncommon
+        Integer id = components.get(DataComponentTypes.RARITY);
+        if (id == null) {
+            id = Rarity.UNCOMMON.ordinal();
+        }
+
         // Use the correct color, determined by the rarity of the item
-        char rarity = Rarity.fromId(components.get(DataComponentTypes.RARITY)).getColor();
+        char rarity = Rarity.fromId(id).getColor();
```

Comment:
thoughts about using `getOrDefault` here instead?

Further, it might be worth it to test what would happen with the following player head:
`/give @a player_head[!rarity,profile={name:"jeb_"}] 1`
Which rarity would vanilla fall back to here?

Reply:
Looks like it falls back to common so I've changed the default to match that